dotcom-rendering/src/web/components/PulsingDot.importable.tsx

```diff
@@ -3,15 +3,16 @@ import { css, keyframes } from '@emotion/react';
 import { storage } from '@guardian/libs';
 import { useEffect, useState } from 'react';
 
-const dotStyles = (colour?: string) => css`
+const dotStyles = (colour?: string, diameter?: number) => css`
```

Comment:
If the diameter needs to be dynamic  - could this be a flag which drives the size of the diameter, rather than the diameter itself?

```
const dotStyles = (colour?: string, large?: boolean) => css..
```


```
width: ${large ? 1 : 0.75}rem
```